x-pack/test/functional/apps/infra/node_details.ts

```diff
@@ -103,42 +103,6 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
   };
 
   describe('Node Details', () => {
-    describe('#Missing fields', function () {
```

Comment:
This test is hard to reproduce now with the no data prompt in the asset details page